src/osd/OSD.cc

```diff
@@ -6208,7 +6209,9 @@ COMMAND("bench " \
 	"osd", "rw")
 COMMAND("flush_pg_stats", "flush pg stats", "osd", "rw")
 COMMAND("heap " \
-	"name=heapcmd,type=CephChoices,strings=dump|start_profiler|stop_profiler|release|stats", \
+	"name=heapcmd,type=CephChoices,strings="\
+	    "dump|start_profiler|stop_profiler|release|(get|set)_release_rate|stats " \
```

Comment:
does ceph_argparse understand the (get|set)_release_rate syntax??

Reply:
sorry, updated.